### PR TITLE
US1D Detect within which building the user is currently  located 

### LINKED ISSUE #11: US1D: As a user, I want to be able to know the building I am currently located in.
The campus and building of current location is always displayed.

### PR DESCRIPTION
**Pull request by sebhani**

### CODE REVIEW

File: components/map/index.js [hunk 1]
```diff
@@ -21,26 +29,43 @@ export default class TheMap extends Component {
   }
 
   render() {
+    // const { coordinate } =
```
Comment (gisellemartel) [inline components/map/index.js:28]: is this commented code needed?

File: components/map/index.js [hunk 2]
```diff
@@ -1,17 +1,25 @@
...
+import { View } from 'react-native';
 import buildings from '../../assets/polygons/polygons';
 import styles from './styles';
 
 export default class TheMap extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      region: {
+      coordinate: {
```
Comment (gisellemartel) [inline components/map/index.js:13]: I don't see this variable being used anywhere, was this intended for future use?

File: components/map/index.js [hunk 3]
```diff
@@ -1,17 +1,25 @@
...
-        longitude: -73.582153
+        longitude: -73.582153,
       },
+      // eslint-disable-next-line react/no-unused-state
+      // region: {
+      //   latitude: 45.492409,
+      //   longitude: -73.582153,
+      //   latitudeDelta: 0.04,
+      //   longitudeDelta: 0.04
+      // },
```
Comment (gisellemartel) [inline components/map/index.js]: Is this commented out code needed?

**Review by gisellemartel — CHANGES_REQUESTED** [review]
Overall, code looks great! I left a few comments concerning some commented code and unused variables that probably should be cleaned up before merging

File: components/map/index.js (hunk 1 above)
Comment (sebhani) [reply components/map/index.js:28]: the map component isn't related to my US. The component that I developed is WithinBuilding component.

Comment (gisellemartel) [reply components/map/index.js:28]: this code was added in one of the commits as a part of this PR, maybe it was added by mistake?

Comment (sebhani) [reply components/map/index.js:28]: maybe because I based the branch on US2D branch.

**Review by gisellemartel — APPROVED** [review]

File: components/withinBuilding/index.js [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+  }
+
+  // returns the name of building the user is currently located in
+  async buildingName() {
+    await this.getCurrentLocation();
+
+    const { location } = this.state;
+
+    // const x = location.coords.longitude;
+    // const y = location.coords.latitude;
```
Comment (eufekt) [inline components/withinBuilding/index.js]: do we need these ?

File: components/map/index.js [hunk 5]
```diff
@@ -48,8 +48,19 @@ export default class TheMap extends Component {
...
+          <MapView.Marker
+            coordinate={{
+              latitude: this.props.updatedRegion.latitude,
+              longitude: this.props.updatedRegion.longitude
+            }}
+            title="title"
+            description="description"
+          />
+          {/* }));
+          })} */}
```
Comment (eufekt) [inline components/map/index.js]: do we need these ?

**Review by eufekt — APPROVED** [review]
The functionality is working. Please resolve the comments and it is good to go.

**Comment by gisellemartel** [comment]
@Nichita-Hariton following your review, requested changes have been made

**Comment by trevornag** [comment]
Nice. Things to consider: Styling this & issue [#18] so as to no overcrowd the map view

**Review by trevornag — APPROVED** [review]

**Pull request merged**